.github/workflows/build.yml

```
@@ -17,13 +17,26 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
-                symfony: ["^5.4", "^6.0"]
+                php: ["8.0", "8.1", "8.2"]
```

we should add PHP 8.3 as well, once we are here. And perhaps even drop 8.0
```suggestion
                php: ["8.1", "8.2", "8.3"]
```